boards/cc2538dk/Makefile.include

```diff
@@ -4,6 +4,7 @@ export CPU_MODEL ?= cc2538nf53
 
 # setup serial terminal
 PORT_LINUX ?= $(firstword $(wildcard /dev/serial/by-id/*XDS100v3*if01* /dev/ttyUSB1))
+PORT_DARWIN ?= $(shell ls -1 /dev/tty.usbserial-* | head -n 1)
```

Comment:
Can you use the `$(firstword $(wildcard …))` pattern, so #4913 does not need to change the file, please?